docs/reference/release-notes/highlights-7.2.0.asciidoc

```diff
@@ -45,3 +43,59 @@ replicated. Indices closed on previous versions of Elasticsearch will
 remain non replicated unless they are opened and closed again in 7.2+.
 
 // end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Geo features in SQL
+beta[] This release introduces the first set of geo features to SQL.
+The implementation is based on the OpenGIS® Implementation Standard for Geographic
+information - "Simple feature access". This is the current de-facto standard for GIS
+system implementation. This release includes a small subset of SQL option AKA ISO 19125-2.
+
+For this initial beta release, we added support for returning
+geo_shapes and geo_points as results, added support for a few geo functions
+(ST_AsText, ST_Distance, ST_GeometryType, ST_GeometryFromText, ST_X, ST_Y, and ST_Z)
+, and added a limited support for using geo_points in distance queries. For example:
+SELECT * FROM my_index WHERE ST_Distance(point, ST_WKTToSQL('point (10 20)')) < 20.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== OpenId Connect authentication realm
+
+This release introduces OpenId Connect as an authentication realm.
+Elasticsearch (with the assistance of Kibana or another web component) can now serve as an
+OpenID Connect Relying Party (RP). {es} supports the Authorization Code Grant and Implicit
+flows as described in http://ela.st/oidc-spec. It also supports consuming and verifying signed ID Tokens
+, RP initiated single sign on (SSO), 3rd party initiated SSO, and RP initiated single logout.
+
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Search as you type field mapping type
+
+The `search_as_you_type` field type is a text-like field optimized to
+provide out-of-the-box support for queries that serve an as-you-type completion
+use case. It creates a series of subfields that are analyzed to index terms
+that can be efficiently matched by a query that partially matches the entire
+indexed text value. Both prefix completion (i.e matching terms starting at the
+beginning of the input) and infix completion (i.e. matching terms at any
+position within the input) are supported.
+
+// end::notable-highlights[]
+
+
+// tag::notable-highlights[]
+[float]
+==== Distance Feature Query
+
+The `distance_feature` query is a specialized query that only works on `date`, `date_nanos`, or `geo_point`
+fields. The query boosts documents scores based on proximity to some given origin.
+For example, you can use this query to give higher scores to documents with dates
+closer to a certain date or locations closer to a certain location.
+closer to a certain date, or to documents with locations closer to a certain location.
+
```

Comment:
Other things I would consider including:
Replicated closed indices
Snapshot repositories app
SQL Median Absolute Deviation
SQL CASE/WHEN/END
HTML Strip Processor

Reply:
Replicated closed indices (already covered)
Snapshot repository app - since this is a Kibana and these are ES release notes probably best for blog 

SQL * and Html Strip Processor - possibly... these are great features, but trying to keep the list relatively small. We have the full release notes which covers these, and IMO the blogs can go into quite a bit more detail.